src/huggingface_hub/constants.py

```diff
@@ -14,6 +14,17 @@
 FLAX_WEIGHTS_NAME = "flax_model.msgpack"
 CONFIG_NAME = "config.json"
 
+FILE_LIST_NAMES = [
+    "config.json",
+    "special_tokens_map.json",
+    "tokenizer_config.json",
+    "tokenizer.json",
+    "vocab.json",
+    "vocab.txt",
+    "merges.txt",
+    "preprocessor_config.json",
+]
```

Comment:
This is incomplete, it's lacking a few files. `added_tokens.json` comes to mind, or the spm-based tokenizer.

Reply:
And the README file